mlflow/server/auth/client.py

```diff
@@ -51,27 +116,129 @@ def get_user(self, username: str):
         return User.from_json(resp["user"])
 
     def update_user_password(self, username: str, password: str):
+        """
+        Update the password of a specific user.
+
+        :param username: The username.
+        :param password: The new password.
+
+        :return: None. Raises ``RestException`` if the user does not exist.
+
+        .. code-block:: python
+            :caption: Example
+
+            import os
+            from mlflow.server.auth.client import AuthServiceClient
+
+            os.environ["MLFLOW_TRACKING_USERNAME"] = "admin"
+            os.environ["MLFLOW_TRACKING_PASSWORD"] = "password"
+
+            client = AuthServiceClient("tracking_uri")
+            client.create_user("newuser", "newpassword")
+
+            client.update_user_password("newuser", "anotherpassword")
+        """
         self._request(
             UPDATE_USER_PASSWORD,
             "PATCH",
             json={"username": username, "password": password},
         )
 
     def update_user_admin(self, username: str, is_admin: bool):
+        """
+        Update the admin status of a specific user.
+
+        :param username: The username.
+        :param is_admin: The new admin status.
+
+        :return: None. Raises ``RestException`` if the user does not exist.
+
+        .. code-block:: python
+            :caption: Example
+
+            import os
+            from mlflow.server.auth.client import AuthServiceClient
+
+            os.environ["MLFLOW_TRACKING_USERNAME"] = "admin"
```

Comment:
Can this be changed as well as the .rst docs so that we're not suggesting users write down their username and password in a script in plaintext?

Reply:
makes sense, I changed them